modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -4,26 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="installation-aws-add-local-zone-locations_{context}"]
-= Opting into AWS Local Zones
+= Opting in to AWS Local Zones
 
 If you plan to create the subnets in AWS Local Zones, you must opt in to each zone group separately.
 
 .Prerequisites
 
 * You have installed the AWS CLI.
-* You have determined into which region you will deploy your {product-title} cluster.
+* You have determined an AWS opt-in Region for where you want to deploy your {product-title} cluster.
 
 .Procedure
 
-. Export a variable to contain the name of the region in which you plan to deploy your {product-title} cluster by running the following command:
+. Export a variable to contain the name of the AWS opt-in Region by running the following command:
 +
 [source,terminal]
 ----
 $ export CLUSTER_REGION="<region_name>" <1>
 ----
-<1> For `<region_name>`, specify a valid AWS region name, such as `us-east-1`.
+<1> For `<region_name>`, specify a valid AWS opt-in Region name, such as `us-east-1`.
```

Comment:
@dfitzmau this `export CLUSTER_REGION` now also became useless as we are using only in `ec2 describe-availability-zones`.
Please remove that export and adjust the command to something like:
```
$ aws --region "<value_of_AWS_Region>" ec2 describe-availability-zones \
    --query 'AvailabilityZones[].[{ZoneName: ZoneName, GroupName: GroupName, Status: OptInStatus}]' \
    --filters Name=zone-type,Values=local-zone \
    --all-availability-zones
```

Reply:
Removed:

_. Export a variable to contain the name of the AWS Region by running the following command:
+
[source,terminal]
----
$ export CLUSTER_REGION="<region_name>" <1>
----
<1> For `<region_name>`, specify a valid AWS Region name, such as `us-east-1`._

Updated the subsequent example.